esf.tf

```diff
@@ -119,6 +119,28 @@ locals {
   ec2 = (length(local.cloudwatch-logs-arns) > 0 ? {
     ec2 = { effect = "Allow", actions = ["ec2:DescribeRegions"], resources = ["*"] } } : {}
   )
+
+  # Unpack release-version into major, minor, patch
+  release-version-unpacked = regexall("^lambda-v([0-9]+)\\.([0-9]+)\\.([0-9]+)$", var.release-version)[0]
```

Comment:
Since by now we have validated the format, we could also just change to:
```
release-version-unpacked = version_parts = split(".", replace(var.release-version, "lambda-v", ""))
```

Just another option instead of regex. Nothing wrong with regex at all, just sharing something maybe more human readable. Either option is fine.

Reply:
Good point! I'm going to use this.